### PR TITLE
Add descriptions for SEO.

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
@gewarren - You're tagged as author for these topics.

This is for [1657405](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1657405).

Please take a look at these changes when you get a chance.

### CODE REVIEW

File: docs/standard/security/securing-state-data.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Securing State Data"
+description: Declare state data as private or internal variables to limit access to it. Such data can still be accessed though reflection, serialization, and in debugging.
```
Comment (gewarren) [inline docs/standard/security/securing-state-data.md]:
```suggestion
description: Declare state data as private or internal variables to limit access to it. Such data can still be accessed through reflection, serialization, and in debugging.
```

File: docs/standard/security/secure-coding-guidelines.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Secure coding guidelines for .NET"
+description: You can design code to work with .NET enforced permissions and other enforcement to help prevent malicious code from accessing data or performing other actions.
```
Comment (gewarren) [inline docs/standard/security/secure-coding-guidelines.md]:
```suggestion
description: Design code to work with .NET-enforced permissions and other enforcement to help prevent malicious code from accessing data or performing other actions.
```

**Review by gewarren — APPROVED** [review]
Approved with a couple suggestions. Thanks for adding these descriptions!

**Comment by TimShererWithAquent** [comment]
#sign-off after implementing changes. 

I don't work in this repo much, so I'll follow up as necessary to get this merged.

**Pull request merged**